saas/api/metrics.py

```diff
@@ -237,24 +237,26 @@ class RevenueMetricAPIView(BeforeMixin, ProviderMixin, APIView):
         }
     """
     def get(self, request, *args, **kwargs):
+        dates = convert_dates_to_utc(month_periods(13, self.ends_at, tz=self.timezone))
+
         # All amounts are in the customer currency.
         account_table, _, _ = \
-            aggregate_monthly_transactions(self.provider,
+            aggregate_transactions_change_by_period(self.provider,
                 Transaction.RECEIVABLE, account_title='Sales',
                 orig='orig', dest='dest',
-                from_date=self.ends_at, tz=self.timezone)
+                date_periods=dates)
 
-        _, payment_amounts = aggregate_monthly(
+        _, payment_amounts = aggregate_transactions_by_period(
             self.provider, Transaction.RECEIVABLE,
             orig='dest', dest='dest',
             orig_account=Transaction.BACKLOG,
             orig_organization=self.provider,
-            from_date=self.ends_at, tz=self.timezone)
+            date_periods=dates[1:])
 
-        _, refund_amounts = aggregate_monthly(
+        _, refund_amounts = aggregate_transactions_by_period(
             self.provider, Transaction.REFUND,
             orig='dest', dest='dest',
-            from_date=self.ends_at, tz=self.timezone)
+            date_periods=dates[1:])
```

Comment:
Following comment on `month_periods(13,...`, this call transforms into `...date_periods=dates)`